### DISCUSSION TITLE
Music pause

### DISCUSSION THREAD

**Discussion by thelift112** [Jcw87/c2-sans-fight#116] [General]
Just found out that you can pause the button with the pause button on headphones, but it unpauses itself if you change the tab, please don't fix the pause if you're not supposed to be able to pause it. Although Black Screen can unpause it.

**Comment by ethanaobrien** [comment]
You could right click the tab and mute the page

**Reply by thelift112** [reply]
But sounds don't get muted if you use headphones to mute it, only the music does, even works at the choose your bad time.
